src/latitudes.cpp

```diff
@@ -204,3 +152,211 @@ double pj_authalic_lat_inverse_exact(double beta, const double *APA,
     }
     return phi;
 }
+#undef PROJ_AUTHALIC_SERIES_VALID
+
+// Compute the coefficients of the trigonometric series for conversions between
+// auxiliary latitudes.  The coefficients are produced by the Maxima code
+// auxlat.mac bundled with GeographicLib.  Load this file into Maxima and run
+// writecppproj()$ -- this produces the 'static const' declarations below.
+// Only a subset of conversions is provided.  To add others, include them in
+// the list "required" in writecppproj().
```

Comment:
This comment confuses me. Specifically the "Load this file ... " part. What is "this" referring to? I get the feeling that this might be copied from somewhere else?